string_test.go

```diff
@@ -74,6 +77,46 @@ func TestUnmarshalString(t *testing.T) {
 	assertNullStr(t, invalid, "invalid json")
 }
 
+func TestStringUnmarshalEasyJSON(t *testing.T) {
+	tests := []struct {
+		data string
+		exp  String
+	}{
+		{
+			data: "null",
+		},
+		{
+			data: `"hat"`,
+			exp:  StringFrom("hat"),
+		},
+		{
+			// For some reason empty string isn't counted as valid. Which is nuts
```

Comment:
This isn't meant to be the case anymore: https://github.com/unravelin/null/pull/1/commits/a9db3ac26fcd2d70230a8dc6286d0c3694700003

Reply:
... and was then overwritten in d46826878c147da6c88e471ea546dde6b1582ad0 :man_shrugging:

Reply:
Merge commit https://github.com/unravelin/null/commit/2e41a9bd154b0127b238e675df2d2a02b93123ae of PR #1 isn't in master. The only way I can imagine that happening is if somebody force pushes to master with outdated changes :worried:

Reply:
```
# The commit with the change.
[icio:~/go/src/github.com/unravelin/null] master ± 
$ git branch --all --contains a9db3ac26fcd2d70230a8dc6286d0c3694700003
  remotes/origin/pr/1

# The merge commit of PR #1.
[icio:~/go/src/github.com/unravelin/null] master ± 
$ git branch --all --contains 2e41a9bd154b0127b238e675df2d2a02b93123ae
error: no such commit 2e41a9bd154b0127b238e675df2d2a02b93123ae
```